src/devices/SenseHat/SenseHatLedMatrixSysFs.cs

```diff
@@ -99,13 +99,9 @@ private void WriteColor(Color color)
             byte g = (byte)(color.G >> 2);
             byte b = (byte)(color.B >> 3);
             ushort col = (ushort)((r << 11) | (g << 5) | b);
-            Span<byte> encoded = stackalloc byte[2]
-            {
-                (byte)(col & 0xff),
-                (byte)(col >> 8)
-            };
 
-            _deviceFile.Write(encoded);
+            _deviceFile.WriteByte((byte)(col & 0xff));
```

Comment:
@hugener I'm fine with not testing this particular fix since this is low risk:
- sysfs is most likely not affected by this
- this is a backup implementation for LED Matrix - I2C is used by default